Source/Scene/Scene.js

```diff
@@ -793,6 +814,142 @@ define([
         }
     });
 
+    function destroyTextures(scene) {
+        scene._colorTexture = scene._colorTexture && !scene._colorTexture.isDestroyed() && scene._colorTexture.destroy();
+        scene._depthStencilTexture = scene._depthStencilTexture && !scene._depthStencilTexture.isDestroyed() && scene._depthStencilTexture.destroy();
+        scene._depthStencilCopyTexture = scene._depthStencilCopyTexture && !scene._depthStencilCopyTexture.isDestroyed() && scene._depthStencilCopyTexture.destroy();
```

Comment:
This is not declared in the constructor function.  Why not name it `_depthStencilGlobeTest`?  "Copy" is too generic, especially as we move towards a post-processing framework.